bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -131,8 +137,21 @@ data:
         REDIS_MASTER_PORT_NUMBER=${REDIS_SENTINEL_INFO[1]}
     fi
 
-    if [[ ! -f /opt/bitnami/redis/etc/replica.conf ]];then
-        cp /opt/bitnami/redis/mounted-etc/replica.conf /opt/bitnami/redis/etc/replica.conf
+    if [[ -n "$REDIS_EXTERNAL_MASTER_HOST" ]]; then
+      REDIS_MASTER_HOST="$REDIS_EXTERNAL_MASTER_HOST"
+      REDIS_MASTER_PORT_NUMBER="${REDIS_EXTERNAL_MASTER_PORT}"
+    fi
+
+    if [[ "$REDIS_REPLICATION_MODE" = "master" ]]; then
+        debug "Starting as master node"
+        if [[ ! -f /opt/bitnami/redis/etc/master.conf ]]; then
+            cp /opt/bitnami/redis/mounted-etc/master.conf /opt/bitnami/redis/etc/master.conf
+        fi
+    else
+        debug "Starting as replica node"
+        if [[ ! -f /opt/bitnami/redis/etc/replica.conf ]];then
+            cp /opt/bitnami/redis/mounted-etc/replica.conf /opt/bitnami/redis/etc/replica.conf
+        fi
     fi
```

Comment:
This is unintentionally reverting https://github.com/bitnami/charts/pull/8781, probably happened during rebase. I'll fix it up.